### ISSUE TITLE
[Feature] Add "Open with..." feature to import files directly from Specter/AirDrop

### ISSUE THREAD

**Issue by w-i-n-s** [Sjors/nthkey-ios#15]
Right now to receive files from anywhere (alternatively to QR scan) user should receive the file and store it locally (in case of not using iCloud sync). And after that user can open it from the app. Possibly, implementing such a feature can increase user experience.
![IMG_6428](https://user-images.githubusercontent.com/2025621/110241360-09e8ca00-7f59-11eb-9e8d-f5680bafe513.PNG)

**Comment by w-i-n-s** [comment]
@Sjors

**Comment by Sjors** [comment]
So this would be for all JSON files? Unlike PSBT, JSON doesn't have magic bytes.

**Comment by w-i-n-s** [comment]
@Sjors Yes, we can allow users to open any JSON file with the app. But we can validate it and if the file isn't the right one or corrupted - the user should receive an alert about it.

**Comment by Sjors** [comment]
Sounds good.
